source/sass/views/projects.scss

```diff
@@ -9,3 +9,7 @@
 #view-projects .btn {
   white-space: initial;
 }
+
+a.pulse-project{
+  color: $black ;
```

Comment:
This is one of those things where it feels like there should be a way to do this without having to write new CSS but I can't think of what that solution would be. So uh, carry on.